src/pwab-samples.ts

```diff
@@ -0,0 +1,163 @@
+import { html, customElement, property, css, LitElement } from "lit-element";
+import * as sampleCode from "./pwab-code";
+
+@customElement("pwab-samples")
+export class pwabsamples extends LitElement {
+  @property({ type: String })
+  theme: string = "lighter";
+
+  @property({ type: String })
+  public code: sampleCode.framework;
+
+  @property({ type: Boolean, attribute: "show-copy" })
+  public showCopyButton: boolean = false;
+
+  @property({ type: String })
+  public color = "#F0F0F0";
+
+  @property({ type: Boolean, attribute: "toolbar" })
+  showToolbar: boolean = false;
+
+  @property({ type: Boolean, attribute: "overlay" })
+  showOverlay: boolean = false;
+
+  textCopied: boolean = false;
+
+  errors: any[] = [];
+
+  constructor() {
+    super();
+  }
+
+  static get styles() {
+    return css`
+      .codeHeader {
+        padding-bottom: 16px;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        z-index: 9999;
+      }
+
+      .codeHeader:not(:first-child) {
+        padding-top: 16px;
+      }
+      .codeHeader h3 {
+        font-family: sans-serif;
+        font-style: normal;
+        font-weight: 600;
+        font-size: 16px;
+        margin: 0;
+        line-height: 24px;
+        padding-left: 1em;
+        width: 60%;
+      }
+
+      .codeHeader div {
+        width: 20em;
+      }
+
+      .pwab-sample {
+      }
+
+      .code-sample {
+        max-height: 600px;
+        margin: 0 16px;
+        overflow: scroll;
+      }
+
+      .copyButton {
+        background: #c5c5c5;
+        color: #3c3c3c;
+        border: none;
+        border-radius: 20px;
+        font-weight: bold;
+        font-size: 12px;
+        padding-top: 3px;
+        padding-bottom: 5px;
+        padding-right: 9px;
+        padding-left: 9px;
+        margin-right: 2em;
+      }
+
+      .fas {
+        font-family: "Font Awesome 5 Free";
+        font-weight: 900;
+        -webkit-font-smoothing: antialiased;
+        display: inline-block;
+        font-style: normal;
+        font-variant: normal;
+        text-rendering: auto;
+        line-height: 1;
+        box-sizing: border-box;
+        font-size: 12px;
+      }
+
+      @media (max-width: 1079px) {
+        .codeHeader h3 {
+          font-size: 12px;
+        }
+      }
+    `;
+  }
+
```

Comment:
https://github.com/pwa-builder/pwab-push/pull/2/commits/6956dfd1c3ca48a22075e48c865cd6c2166d67a0

https://github.com/pwa-builder/pwab-push/pull/2/commits/ec96aa0449de75554685044f0d7666ebe0933e68

Removed monaco editor code and lifestyle changes, preserved in history.